app/indexers/oregon_digital/metadata_indexer.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module OregonDigital
+  # OVERRIDE FROM HYRAX TO ADD OUR OWN FIELDS
+  class MetadataIndexer < ActiveFedora::RDF::IndexingService
```

Comment:
Could this be simplified by inheriting from `Hyrax::BasicMetadataIndexer` then just overriding the bits that we actually want to change? Is this better or worse practice in Ruby/Rails land?